### PR TITLE
Rebrand and update lightstep docs

### PR DESCRIPTION
**Pull request by robswhitmore**
Please provide a description for what this PR is for.

Lightstep has been rebranded to Cloud Observability from ServiceNow. Additionally, we now use the OpenTelemetry Collector to ingest metrics, traces, and logs from third-parties. These updates include both changes.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ X ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by robswhitmore** [comment]
I'd like to also add the label `cherrypick/release-1.19` but I don't seem to be able to edit the labels field.

**Comment by ericvn** [comment]
Depending on when this merges, it may or may not need the 1.19 label as there is no 1.19 branch (right now). The branch will be created when 1.19.0 is release.d.

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by robswhitmore** [comment]
I fixed the lint errors that I can, but the errors for spacing around lists can't be fixed because those are inside yaml code blocks. What should I do?

Also, what should we tag this PR so that it goes into the latest release?

**Comment by ericvn** [comment]
@istio/wg-policies-and-telemetry-maintainers Can you review this as owners of the page as well. Thanks.

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 1]
```diff
@@ -1,197 +1,237 @@
...
+              level: "debug"
+          pipelines:
+            metrics:
+              receivers: [prometheus]
+              processors: [batch]
+              exporters: [logging,otlp]
     {{< /text >}}
 
-### Deploy Istio with Public or Developer Mode Satellites
+2. Create an `otel-collector-deployment.yaml` file by copying the following code.
```
Comment (ericvn) [inline content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:80]:
```suggestion
1. Create an `otel-collector-deployment.yaml` file by copying the following code.
```

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 2]
```diff
@@ -1,197 +1,237 @@
...
-    {{< image link="./istio-explorer.png" caption="Explorer view" >}}
+    Verify that the OpenTelemetry Collector is running:
 
-1.  Find the query bar at the top. The query bar allows you to interactively filter results by a **Service**, **Operation**, and **Tag** values.
+    {{< text plain >}}
+    kubectl get pods -l app=otel-collector
+    {{< /text >}}
 
-1.  Select `productpage.default` from the **Service** drop-down list.
+2. Apply the ConfigMap to your Kubernetes cluster
```
Comment (ericvn) [inline content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:212]:
```suggestion
1. Apply the ConfigMap to your Kubernetes cluster
```

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 3]
```diff
@@ -1,197 +1,237 @@
...
+    Verify that the ConfigMap has been created
 
-1.  Click on the first row in the table of example traces below the latency histogram to see the details
-    corresponding to your refresh of the `/productpage`. The page then looks similar to:
+    {{< text plain >}}
+    kubectl get configmap otel-collector-conf
+    {{< /text >}}
 
-    {{< image link="./istio-tracing-details-lightstep.png" caption="Detailed Trace View" >}}
+3. Apply the Secret to your Kubernetes cluster
```
Comment (ericvn) [inline content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:224]:
```suggestion
1. Apply the Secret to your Kubernetes cluster
```

**Review by ericvn — COMMENTED** [review]
I believe these 3 suggestions will fix the numbering. Looks like the line numbering is off by 1.

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 4]
```diff
@@ -1,197 +1,237 @@
...
+    kubectl apply -f lightstep-secret.yaml
+    {{< /text >}}
 
-Two spans in the trace represent every RPC. For example, the call from `productpage` to `reviews` starts
-with the span labeled with the `reviews.default.svc.cluster.local:9080/*` operation and the
-`productpage.default: proxy client` service. This service represents the client-side span of the call. The screenshot shows
-that the call took 15.30 ms. The second span is labeled with the `reviews.default.svc.cluster.local:9080/*` operation
-and the `reviews.default: proxy server` service. The second span is a child of the first span and represents the
-server-side span of the call. The screenshot shows that the call took 14.60 ms.
+## View metrics in Cloud Observability
```
Comment (lei-tang) [inline content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:230]: The folder content/en/docs/tasks/observability/distributed-tracing/lightstep is for tracing. Consider placing the metric guide into a different folder.

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 5]
```diff
@@ -1,197 +1,237 @@
...
-keywords: [telemetry,tracing,lightstep]
+keywords: [telemetry,tracing,lightstep, servicenow, cloud observability]
 aliases:
  - /docs/tasks/telemetry/distributed-tracing/lightstep/
 owner: istio/wg-policies-and-telemetry-maintainers
 test: no
 ---
 
-{{< boilerplate telemetry-tracing-tips >}}
+You use the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) to send Istio metrics to Cloud Observability using Istio proxies within each pod. Metrics are collected and forwarded to the OpenTelemetry Collector, which acts as a central collection and processing point and then sends those metrics to Cloud Observability.
```
Comment (lei-tang) [inline content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:12]: The folder content/en/docs/tasks/observability/distributed-tracing/lightstep is for tracing. Consider placing the metric guide into a different folder.

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md (hunk 4 above)
Comment (robswhitmore) [reply content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:230]: Moved to the metrics section

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md (hunk 5 above)
Comment (robswhitmore) [reply content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:12]: Moved to the metrics section

Comment (robswhitmore) [reply content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md]: Removed references to Lightstep in site where it was mentioned in the context of tracing (this also fixed broken links caused when moving the topics from Tracing to Metrics). There is still a broken link in `/news/releases/1.1.x/announcing-1.1/change-notes/`, but I don't think I should change that, right?

**Comment by robswhitmore** [comment]
I don't think I can fix this link because it's in a news post, right?

**Comment by robswhitmore** [comment]
@lei-tang or whoever else might be able to help -
The linter is failing with a broken link to an old change note. I don't think I should update this, correct?
image

**Comment by lei-tang** [comment]
@ericvn Do you have suggestions for fixing the broken link error above?

File: content/en/docs/tasks/observability/metrics/lightstep/index.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
+---
+title: Viewing Istio Metrics in Cloud Observability from ServiceNow
+description: This task shows you how to use OpenTelemetry to ingest and view Istio metrics in Cloud Observability (formerly Lightstep).
+weight: 11
+keywords: [telemetry, metrics ,lightstep, servicenow, cloud observability]
+aliases:
+ - /docs/tasks/observability/metrics/lightstep/
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/lightstep/index.md:7]: Why do you have this alias here? It's just pointing to itself.

**Review by ericvn — DISMISSED** [review]

**Comment by ericvn** [comment]
Lint error is Running linkinator...
  [404] http://localhost:5080/docs/tasks/observability/distributed-tracing/lightstep/
ERROR: Detected 1 broken links. Scanned 5787 links in 39.311 seconds.

You removed this file so makes sense that the link isn't found.

What do you want people who pointed to it to point to now?  You could point to an older releases version of the file by changing the URL to point to a specific older release.  Maybe you want to add an alias to a new page so the page would then be used in the other pages place (ie. replace the alias I've asked you to remove with `/docs/tasks/observability/distributed-tracing/lightstep/`).

**Comment by ericvn** [comment]
Also added the cherry-pick label since we have cut the new branch.

**Comment by robswhitmore** [comment]
@ericvn I did what you suggested and changed the alias to the page I deleted (/docs/tasks/observability/distributed-tracing/lightstep/), but I'm still getting the same broken link and now an additional broken link for that alias. I followed the directions [here](/docs/tasks/observability/distributed-tracing/lightstep/), but I must be missing something?

**Comment by dhawton** [comment]
> @ericvn I did what you suggested and changed the alias to the page I deleted (/docs/tasks/observability/distributed-tracing/lightstep/), but I'm still getting the same broken link and now an additional broken link for that alias. I followed the directions [here](/docs/tasks/observability/distributed-tracing/lightstep/), but I must be missing something?

The lint report tells you the file the link is originating from, which is https://github.com/robswhitmore/istio.io/blob/robswhitmore/rebrand-update-lightstep/content/en/news/releases/1.1.x/announcing-1.1/change-notes/index.md?plain=1#L165

Since you're changing the name of the file, link to an archived version of the page, for example: https://istio.io/v1.18/docs/tasks/observability/distributed-tracing/lightstep/

**Comment by ericvn** [comment]
@robswhitmore Daniel is correct. There are two ways to fix the issue. One is to update the link which is pointing to the file you removed in `news/releases/1.1.x/announcing-1.1/change-notes/` to point to and older existing version of the page (use 1.19 or 1.18 for example). You can look at a commit where I did that yesterday: https://github.com/istio/istio.io/commit/fbb5c8df9bf9f0e6969a4413f0a669c69b62042b prefixing `https://archive.istio.io/v1.17` to the /docs/... that was removed. In this case, the link will point to the same information as before.

As I noted, if the new page you replaced the deleted page with is an acceptable alternative, then you can simply add `[/docs/tasks/observability/distributed-tracing/lightstep/](https://istio.io/v1.18/docs/tasks/observability/distributed-tracing/lightstep/)` to the aliases list on the new page, and when they click the link it will take them to the new page.

Pick whichever way yields the best information to the reader.

File: content/en/docs/tasks/observability/metrics/lightstep/index.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
+---
+title: Viewing Istio Metrics in Cloud Observability from ServiceNow
+description: This task shows you how to use OpenTelemetry to ingest and view Istio metrics in Cloud Observability (formerly Lightstep).
+weight: 11
+keywords: [telemetry, metrics ,lightstep, servicenow, cloud observability]
+aliases:
+ - /docs/tasks/observability/distributed-tracing/lightstep/
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/lightstep/index.md:7]: I'm wondering if there is a format issue with number of leading spaces, as this does look correct. Need to try pulling your PR and trying the link-check locally.
```suggestion
    - /docs/tasks/observability/distributed-tracing/lightstep/
```

Comment (ericvn) [reply content/en/docs/tasks/observability/metrics/lightstep/index.md:7]: Basically, the alias discussion is here: https://istio.io/latest/docs/releases/contribute/front-matter/#rename-move-or-delete-pages

File: content/en/docs/tasks/observability/metrics/lightstep/index.md [hunk 8]
```diff
@@ -0,0 +1,232 @@
+---
+title: Viewing Istio Metrics in Cloud Observability from ServiceNow
+description: This task shows you how to use OpenTelemetry to ingest and view Istio metrics in Cloud Observability (formerly Lightstep).
+weight: 11
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/lightstep/index.md:4]: Also the weight here is used to set the order of the links within the page. You can look at your preview at https://deploy-preview-13779--preliminary-istio.netlify.app/latest/docs/tasks/observability/metrics/ to see if the orderings correct. Higher numbers go down the order.

File: content/en/docs/tasks/observability/metrics/lightstep/index.md (hunk 7 above)
Comment (ericvn) [reply content/en/docs/tasks/observability/metrics/lightstep/index.md:7]: I did pull your branch locally and try a few things with the aliases, and I never got it to work. I guess I would just do option 1 and update the 1 link to point to the archive and remove the alias.

Comment (ericvn) [reply content/en/docs/tasks/observability/metrics/lightstep/index.md:7]: @frankbu is there some magic on the `aliases:`?

**Comment by robswhitmore** [comment]
@ericvn and @lei-tang - my commit finally passed the lint check, so I think it's good to go. Thanks for all your help!

File: content/en/docs/tasks/observability/metrics/lightstep/index.md [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+              mountPath: /conf
+          volumes:
+          - configMap:
+              name: otel-collector-conf
+            name: otel-collector-config-vol
+    {{< /text >}}
+
+## Configure Istio Operator
+
+Create an `istio-operator.yaml` file by copying the following code.
```
Comment (lei-tang) [inline content/en/docs/tasks/observability/metrics/lightstep/index.md:123]: Is 'istio-operator.yaml' here used in the guide? A quick search did not return any results.

Comment (robswhitmore) [reply content/en/docs/tasks/observability/metrics/lightstep/index.md:123]: I'm not sure what you mean by "used in the guide?" Do you mean elsewhere in Istio docs? It's used in this topic to configure the k8s operator for Istio.

Comment (lei-tang) [reply content/en/docs/tasks/observability/metrics/lightstep/index.md:123]: > "It's used in this topic to configure the k8s operator for Istio."
Which command lines in this guide use 'istio-operator.yaml'?

Comment (robswhitmore) [reply content/en/docs/tasks/observability/metrics/lightstep/index.md:123]: Thanks - I realized what he was asking and agreed, so I sent it back to the engineering team. They are working on updates.Thanks for catching that - I definitely should have, but with this rebrand, I've been 🤪!

**Comment by ericvn** [comment]
Does https://github.com/istio/istio.io/pull/13996 replace this?

**Comment by robswhitmore** [comment]
News to me! 😕  Let me find out what's going on...

**Comment by robswhitmore** [comment]
Looks like the eng team decided to focus on Tracing, so that new PR is separate from this one, and is taking precedence. Sorry for all the confusion!

**Comment by ericvn** [comment]
> Looks like the eng team decided to focus on Tracing, so that new PR is separate from this one, and is taking precedence. Sorry for all the confusion!

I think you are saying that at some point this PR will be revisited.

**Comment by robswhitmore** [comment]
Yes, at some point. Thanks -

**Comment by craigbox** [comment]
We've decided to remove this from the Istio docs, as Lightstep just takes traces in OTLP format and there's nothing specific to add above and beyond that.

We'll link https://docs.lightstep.com/integrations/ingest-metrics-istio from the OTLP docs to make it clear what people need to do.

**Pull request closed without merging**